app/hub/Views/UpgradePlanView/UpgradePlanView.jsx

```diff
@@ -0,0 +1,958 @@
+/**
+ * Upgrade Plan View Component
+ *
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2019 Ghostery, Inc. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import React, { useRef } from 'react';
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import globals from '../../../../src/classes/Globals';
+
+/**
+ * A React class component for rendering the Home View
+ * @return {JSX} JSX for rendering the Home View of the Hub app
+ * @memberof HubComponents
+ */
```

Comment:
I think this comment might need an edit :)

Reply:
Edited!